Standards/scs-XXXX-v1-security-of-iaas-service-software.md

```diff
@@ -0,0 +1,138 @@
+| SLURP               | Skip Level Upgrade Release Process - A Process that allows upgrades between two releases, while skipping the one in between them.        |
+| OSSN                | [OpenStack Security Note](https://wiki.openstack.org/wiki/Security_Notes) - security issues from 3rd parties or due to misconfigurations. |
+| OSSA                | [OpenStack Security Advisories](https://security.openstack.org/ossalist.html) - security issues and advices for OpenStack.               |
+
+## Motivation
+
+On the IaaS Layer the software, that needs to be considered in the scope of this standard, is mainly the APIs of IaaS Services.
+Also there might be shared libraries and other dependencies, that could be considered part of the IaaS Layer.
+In software projects like e.g. OpenStack that provide the main services and all APIs, the software will be modified and receive bug fixes continuously and will receive releases of new versions on a regular basis.
+Older releases will at some point not receive updates anymore, because maintaining more and more releases simultaneously requires too much manpower.
+Thus older versions will also eventually not receive security updates anymore.
+Using versions which do not receive updates anymore threatens the baseline security of deployments and should be avoided under all circumstances.
+
+## Design Considerations
+
+It would be possible to define a minimum version of IaaS Layer software to avoid security risks.
+In the following paragraphs several options of defining a minimum version or dealing with security patches otherwise are discussed.
+
+### Options considered
+
+#### Only Allow the current versions of Software
+
+Considering that OpenStack as one provider of IaaS Layer Software has two releases per year, with one SLURP release per year, this option would require CSPs to update their deployment once or twice a year.
+Updating a whole deployment is a lot of work and requires also good life-cycle management.
+Following only the SLURP releases would reduce this work to once per year.
+
+While following new releases closely already provides a deployment with recent bug fixes and new features, it also makes developing standards easier.
+Differences between releases will accumulate eventually and may render older releases non-compliant to the SCS standards at some point.
+
+On the other hand on the IaaS Level there aren't many breaking changes introduced by releases and also most standards will also work with older releases.
+Security Updates and Bug fixes are also provided by OpenStack for a few older releases with the state `maintained`.
+Additionally the [SCS reference implementation](https://github.com/SovereignCloudStack/release-notes/blob/main/Release7.md) is integrating OpenStack releases after half a year - so about the time when a new release is cut by OpenStack.
```

Comment:
```suggestion
Additionally the [SCS reference implementation](https://github.com/SovereignCloudStack/release-notes/blob/main/Release7.md) is integrating OpenStack releases after half a year - so about the time when a new release is published by OpenStack.
```

Did you mean something like "published"? "cut" sounds more like discontinued I think.